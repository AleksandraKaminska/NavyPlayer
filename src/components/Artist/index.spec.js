import React from 'react'
import { BrowserRouter as Router } from 'react-router-dom'
import { Artist } from './index'
import renderer from 'react-test-renderer'

describe('Artist', () => {
  it('should render self', () => {
    const rendered = renderer.create(
      <Router>
        <Artist artist={{}} />
      </Router>
    )
    expect(rendered.toJSON()).toMatchSnapshot()
  })
})
