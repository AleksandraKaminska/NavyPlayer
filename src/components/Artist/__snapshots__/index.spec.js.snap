// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Artist should render self 1`] = `
<section
  className="artist__info"
>
  <h1
    className="artist__info__name"
  >
    <a
      aria-current={null}
      className="artist__info__listen"
      href="/artist/undefined"
      onClick={[Function]}
    >
      <button
        onClick={[Function]}
      >
        Listen
      </button>
    </a>
  </h1>
  <p
    className="artist__info__bio"
  >
    
  </p>
  <div
    className="artist__info__genres"
  />
</section>
`;
